docs/spec/callables.rst

```diff
@@ -571,10 +569,10 @@ of keyword-only parameters is ignored for purposes of subtyping::
         f1: KwOnly = standard  # OK
         f2: Standard = kw_only  # Error
 
-Likewise, callable ``A`` is a subtype of callable ``B`` if all positional-only
-parameters in ``B`` are present in ``A`` as either positional-only parameters
-or standard (positional or keyword) parameters. The names of positional-only
-parameters are ignored for purposes of subtyping::
+Likewise, callable ``B`` is assignable to callable ``A`` only if all
+positional-only parameters in ``A`` are present in ``B`` as either
+positional-only parameters or standard (positional or keyword) parameters. The
```

Comment:
Pre-existing issue, but I'd prefer to give the formal definition here before the informal understanding:

```suggestion
positional-only parameters or positional-or-keyword (standard) parameters. The
```

Reply:
That's reasonable, but I really do want to avoid this PR growing and growing indefinitely as a catch-all for unrelated tweaks to existing text, so I'm going to leave out this change.